paddle/fluid/pybind/imperative.cc

```diff
@@ -582,26 +582,82 @@ void BindImperative(py::module *m_ptr) {
               py::object &value_obj) {
              auto self_tensor =
                  self->MutableVar()->GetMutable<framework::LoDTensor>();
-             auto self_numpy = TensorToPyArray(*self_tensor);
+             PyObject *index_ptr = !PyTuple_Check(_index.ptr())
```

Comment:
Not Important, you can choose to change or not

Why add the "!"? Can we write simpler (prefer shorter, positive condition):

"PyTuple_Check(_index.ptr()) ? _index.ptr(): PyTuple_Pack(1, _index.ptr())"

Reply:
Thanks, I will modify it next PR.